frontend/src/mocks/handlers/team.ts

```diff
@@ -0,0 +1,11 @@
+import { rest } from 'msw';
+import { teamPlaces as teamPlaceData } from '~/mocks/fixtures/team';
+
+const teamPlaces = { teamPlaceData };
+
+export const teamHandlers = [
+  // 팀플레이스 목록 조회
+  rest.get('/api/me/team-places', async (req, res, ctx) => {
+    return res(ctx.status(200), ctx.json(teamPlaces));

```

Comment:
```
{
		"teamPlaces" : [
				{
						"id": 1,
						"displayName": "우리의 팀플레이스",
						"teamPlaceColor": 1
				},
				...
		]
}
```
이렇게 오기 때문에 얘는 한번 감싸서 내보내야하지 않나?

```suggestion
    return res(ctx.status(200), ctx.json({teamPlaces}));

```

Reply:
스크린샷 올렸듯이 괄호 쳐지는지 안 쳐지는지는 확인하고 올린 것은 맞는데, 이제 보니까 `teamPlaceData` 같은 잘못된 키 이름을 넣어버린 게 문제였다 ㅋㅋ 미안해...

이거 올바르게 고치고 다시 리뷰요청할게

Reply:
![image](https://github.com/woowacourse-teams/2023-team-by-team/assets/87642422/5f427fbc-2d3e-4b17-ad87-d3fe5d567f39)

고쳤어! 이제 문제 없는 것 같아
이상한 부분 아직도 있으면 알려주면 고맙겠으

Reply:
나는 근데 괄호를 위에서 치는것 보다 return 할 때 감싸는게 좋아보여. 다른 것을 이미 그렇게 해서 일관성이 떨어지기도 하구 + 나중에 고쳐야할 때 헷갈릴것같아!